examples/camera-app/linux/src/camera-device.cpp

```diff
@@ -644,6 +647,31 @@ CameraError CameraDevice::SetMicrophoneVolume(uint8_t microphoneVol)
     return CameraError::SUCCESS;
 }
 
+int16_t CameraDevice::GetPanMin()
+{
+    return -90;
+}
+
+int16_t CameraDevice::GetPanMax()
```

Comment:
Are those return values configurable? we should use constexpr to define them within the anonymous namespace instead of hard coding, such as

```
namespace {

// Constants
constexpr uint32_t kSessionTimeoutSeconds = 30;

} // namespace
```